druid/src/event.rs

```diff
@@ -131,9 +116,31 @@ pub enum Event {
     /// [`Widget`]: trait.Widget.html
     /// [`EventCtx::submit_command`]: struct.EventCtx.html#method.submit_command
     Command(Command),
-    /// A command still in the process of being dispatched. This is an internal
-    /// event and should generally not be handled directly by widgets, but is
-    /// important for containers to dispatch to their children.
+    /// Internal druid event.
+    ///
+    /// This should always be passed down to descendant [`WidgetPod`]s.
+    ///
+    /// [`WidgetPod`]: struct.WidgetPod.html
+    Internal(InternalEvent),
+}
+
+/// Internal events used by druid inside [`WidgetPod`].
+///
+/// These events are translated into regular [`Event`]s
+/// and should not be used directly.
+///
+/// [`WidgetPod`]: struct.WidgetPod.html
+/// [`Event`]: enum.Event.html
+#[derive(Debug, Clone)]
+pub enum InternalEvent {
+    /// Sent to the root widget when the window size changes.
+    Size(Size),
```

Comment:
There's an ongoing question about whether or not this should actually be more widely used, and sent down the tree by default; and even if not I do think of it as public API, custom widgets would regularly want to handle it I think?